src/deepsparse/open_pif_paf/schemas.py

```diff
@@ -33,6 +35,26 @@ class OpenPifPafInput(ComputerVisionSchema):
     pass
 
 
+class OpenPifPafFields(BaseModel):
+    """
+    Open Pif Paf is composed of two stages:
+     - Computing Cif/Caf fields using a parametrized model
+     - Applying a matching algorithm to obtain the final pose
+        predictions
+    In some cases (e.g. for validation), it may be useful to
+    obtain the Cif/Caf fields as output.
+    """
+
+    fields: List[List[numpy.ndarray]] = Field(
```

Comment:
Oooo, our old friend `numpy.ndarray`; will this cause issues with serialization needed for deepsparse server? Do we need to add `Any`?

Reply:
Yeah, was thinking about that.  
We can now assume that this pathway is not indented to be used broadly by the users, so we do not need the support for it in the server. Weak assumption, I know, but we can still change this in the future.

Actually, @corey-nm didn't you find some sweet hack back in the days to enable numpy serialization?

Reply:
let's see if we can get this resolved, if not, let's add an inline todo

Reply:
Resolved

Reply:
What was the resolution?

Reply:
If we call the openpifpaf pipeline that is set to output cifcaf in the server, ValueError will be thrown.